examples/haas.cfg

```diff
@@ -114,3 +114,7 @@ haas.ext.auth.database =
 # are required.
 vlans = 100-200
 
+[haas.ext.switches.dell]
+# Make modifications to the dell switches' configuration persistent
```

Comment:
@henn, what you're suggesting is inconsistent with what most of the comments in this file do. See e.g. the comments in the database section. I think it's clear as is.